libexec/scoop-update.ps1

```diff
@@ -98,6 +98,16 @@ function update_scoop() {
         $isRepoChanged = !($currentRepo -match $configRepo)
         $isBranchChanged = !($currentBranch -match "\*\s+$configBranch")
 
+        # Stash uncommitted changes
+        if (git -C "$currentdir" diff HEAD --name-only) {
+            if (get_config autostash_on_conflict) {
+                warn "Uncommitted changes detected. Stashing..."
+                git -C "$currentdir" stash push -m "WIP at $([System.DateTime]::Now.ToString('o'))" -u -q
+            } else {
+                abort "Uncommitted changes detected. Updating Aborted."
```

Comment:
```suggestion
                warn "Uncommitted changes detected. Update aborted."
                return
```

`abort` will exit the entire script, which we don't want

Reply:
Message from Copilot lol.